tools/lsp/preview.rs

```diff
@@ -1016,6 +1016,12 @@ pub enum LoadBehavior {
     Load,
 }
 
+#[derive(Copy, Clone)]
+pub enum WindowBehavior {
+    WindowUnchanged,
+    BringWindowToFront,
+}
```

Comment:
I think this can be merge with the LoadBehavior.
It should be

```
pub enum LoadBehavor {
   Reload,
   OpenWindow, // formelly Load
   BringWindowToFront
}
```

Not even sure we need both. As LoadBehavior::Load had already the meaning of showing the window. (And the location we use it where we don't want to bring the window to the front is cases we know it already has the focus, and Reload could be used instead.

Reply:
Excellent idea. Done.